### PR TITLE
Added copyright test setup.sh

### LINKED ISSUE #77: Migrate development acceptance tests to Haskell
Development acceptance tests should be ran with

```sh
stack test
```

### LINKED ISSUE #78: Migrate AcceptanceTest.bat
First 3 steps are in test/Makefile

### LINKED ISSUE #79: Migrate AcceptanceCheck.bat
Just triggers DatChecker on the output file

### PR DESCRIPTION
**Pull request by keremispirli**
Adds FixCopyrightNotices.bat
Removes migrated files

Resolves #78 
Closes #79 
Closes #77

### CODE REVIEW

**Comment by keremispirli** [comment]
See [this build](https://semaphoreci.com/capitanbatata/torxakis/branches/feat-testcopyright/builds/11) for a sample build that is broken due to missing copyright notice. Error code informs you about how to fix the problem.

If you add any new files, just run this command before pushing:

`$TorXakisRepo>./test/copyright/FixCopyrightNotices.bat .`

Sample output:
```
Setting 'solve' flag
Searching for Copyright notice:
TorXakis - Model Based Testing
Copyright (c) 2015-2017 TNO and Radboud University
See LICENSE at root directory of this repository.

.\.torxakis.yaml fixed!
#Files = 289
#Violations = 1
Fixed
```

File: test/copyright/Copyright.java [hunk 1]
```diff
@@ -9,187 +9,204 @@
 import java.util.*;
 
 public class Copyright {
```
Comment (capitanbatata) [inline test/copyright/Copyright.java]: BTW, why not making this class `static`?

File: test/copyright/Copyright.java [hunk 2]
```diff
@@ -9,187 +9,204 @@
...
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
```
Comment (capitanbatata) [inline test/copyright/Copyright.java]: :scream: those are a lot of braces :stuck_out_tongue:

File: test/datcheck/DatChecker.java [hunk 3]
```diff
@@ -1,261 +0,0 @@
-/*
```
Comment (capitanbatata) [inline test/datcheck/DatChecker.java]: :champagne: bye bye datchecker

File: test/copyright/Copyright.java (hunk 1 above)
Comment (keremispirli) [reply test/copyright/Copyright.java]: It's an isolated tool, so I didn't bother rewriting everything.

File: test/datcheck/DatChecker.java (hunk 3 above)
Comment (keremispirli) [reply test/datcheck/DatChecker.java]: 🎉 🎉 🎉

**Pull request merged**